src/frontend/Sider/Personoversikt/Aktivitetsoversikt/Aktivitetsoversikt.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+import Aktiviteter from './Aktiviteter';
+import { Aktivitet } from './typer';
+import { useApp } from '../../../context/AppContext';
+import DataViewer from '../../../komponenter/DataViewer';
+import { byggHenterRessurs, Ressurs } from '../../../typer/ressurs';
+
+const Aktivitetsoversikt: React.FC<{ fagsakPersonId: string }> = ({ fagsakPersonId }) => {
+    const { request } = useApp();
+
+    const [aktiviteter, settAktiviteter] = useState<Ressurs<Aktivitet[]>>(byggHenterRessurs());
+
+    useEffect(() => {
+        request<Aktivitet[], null>(`/api/sak/aktivitet/${fagsakPersonId}`, 'GET').then(
+            settAktiviteter
+        );
+    }, [request, fagsakPersonId]);
+
+    return (
+        <DataViewer response={{ aktiviteter }}>
+            {({ aktiviteter }) => (
+                <>
+                    <Aktiviteter aktiviteter={aktiviteter} />
+                </>
```

Comment:
Er det nødvendig med tomme fragments her?

Reply:
Hvis det er fordi det kommer hovedytelser etterhvert så kan det også hende at når det skjer så trenger man en container for spacing osv også, så det kan legges inn ved behov